diff --git a/.circleci/config.yml b/.circleci/config.yml
index 846811499..130ce8471 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -455,32 +455,14 @@ references:
   bucket_upload_jobs: &bucket_upload_jobs
     - Setup Environment:
         context: bucket_upload_env
-    - Run Unit Testing And Lint:
-        context: bucket_upload_env
-        requires:
-          - Setup Environment
-    - Run Validations:
-        context: bucket_upload_env
-        requires:
-          - Setup Environment
     - Create Instances:
         context: bucket_upload_env
         requires:
-          - Run Unit Testing And Lint
-          - Run Validations
-    - Install Packs in Server 6_0:
-        context: bucket_upload_env
-        requires:
-          - Create Instances
-    - Install Packs in Server Master:
-        context: bucket_upload_env
-        requires:
-          - Create Instances
+          - Setup Environment
     - Upload Packs To Marketplace:
         context: bucket_upload_env
         requires:
-          - Install Packs in Server 6_0
-          - Install Packs in Server Master
+           - Create Instances
 
   check_user_permissions_to_upload_packs: &check_user_permissions_to_upload_packs
     when:
@@ -1000,7 +982,6 @@ jobs:
       - *attach_workspace
       - *add_ssh_keys
       - *prepare_environment
-      - *check_user_permissions_to_upload_packs
       - run:
           name: Force Upload Packs To Marketplace Storage
           command: |
@@ -1013,9 +994,10 @@ jobs:
             echo $GCS_MARKET_KEY > $GCS_PATH
 
             GCS_BUILD_BUCKET="marketplace-ci-build"
-            GCS_MARKET_BUCKET="marketplace-dist"
+            GCS_MARKET_BUCKET="marketplace-dist-dev"
+            STORAGE_BATH_PATH="upload-flow/builds/$CIRCLE_BRANCH/$CREATE_INSTANCES_JOB_NUMBER/content/packs"
 
-            python3 ./Tests/Marketplace/copy_and_upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -pb "$GCS_MARKET_BUCKET" -bb "$GCS_BUILD_BUCKET" -s $GCS_PATH -n $CREATE_INSTANCES_JOB_NUMBER -c $CIRCLE_BRANCH -p "${PACKS_TO_UPLOAD}"
+            python3 ./Tests/Marketplace/copy_and_upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -pb "$GCS_MARKET_BUCKET" -bb "$GCS_BUILD_BUCKET" -s $GCS_PATH -n $CREATE_INSTANCES_JOB_NUMBER -c $CIRCLE_BRANCH -p "${PACKS_TO_UPLOAD}" -pbp "${STORAGE_BATH_PATH}"
             rm $GCS_PATH
           when: on_success
       - *store_artifacts
@@ -1028,11 +1010,10 @@ jobs:
       - *attach_workspace
       - *add_ssh_keys
       - *prepare_environment
-      - *check_user_permissions_to_upload_packs
       - when:
           condition:
             equal:
-              - master
+              - upload-metadata
               - << pipeline.git.branch >>
           steps:
             - run:
@@ -1047,9 +1028,10 @@ jobs:
                   echo $GCS_MARKET_KEY > $GCS_PATH
 
                   GCS_BUILD_BUCKET="marketplace-ci-build"
-                  GCS_MARKET_BUCKET="marketplace-dist"
+                  GCS_MARKET_TESTING_BUCKET="marketplace-dist-dev"
+                  SOURCE_TESTING_PATH="upload-flow/builds/$CIRCLE_BRANCH/$CREATE_INSTANCES_JOB_NUMBER/content/packs"
 
-                  python3 ./Tests/Marketplace/copy_and_upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -pb "$GCS_MARKET_BUCKET" -bb "$GCS_BUILD_BUCKET" -s $GCS_PATH -n $CREATE_INSTANCES_JOB_NUMBER -c $CIRCLE_BRANCH
+                  python3 ./Tests/Marketplace/copy_and_upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -pb "$GCS_MARKET_TESTING_BUCKET" -bb "$GCS_BUILD_BUCKET" -s $GCS_PATH -n $CREATE_INSTANCES_JOB_NUMBER -c $CIRCLE_BRANCH -pbp "$SOURCE_TESTING_PATH"
                   rm $GCS_PATH
             - run:
                 name: Validate Premium Packs
@@ -1315,7 +1297,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - upload-metadata
           requires:
             - Setup Environment
       - Force Upload Packs To Marketplace:
@@ -1323,7 +1305,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - upload-metadata
           requires:
             - Create Instances
 
diff --git a/Tests/Marketplace/marketplace_services.py b/Tests/Marketplace/marketplace_services.py
index 6ebb4d11a..14a9a817c 100644
--- a/Tests/Marketplace/marketplace_services.py
+++ b/Tests/Marketplace/marketplace_services.py
@@ -59,9 +59,9 @@ class GCPConfig(object):
     STORAGE_CONTENT_PATH = "content"  # base path for content in gcs
     USE_GCS_RELATIVE_PATH = True  # whether to use relative path in uploaded to gcs images
     GCS_PUBLIC_URL = "https://storage.googleapis.com"  # disable-secrets-detection
-    PRODUCTION_BUCKET = "marketplace-dist"
+    PRODUCTION_BUCKET = "marketplace-dist-dev"
     CI_BUILD_BUCKET = "marketplace-ci-build"
-    PRODUCTION_PRIVATE_BUCKET = "marketplace-dist-private"
+    PRODUCTION_PRIVATE_BUCKET = "marketplace-dist-private-test"
     CI_PRIVATE_BUCKET = "marketplace-ci-build-private"
     BASE_PACK = "Base"  # base pack name
     INDEX_NAME = "index"  # main index folder name
diff --git a/Tests/scripts/prepare_content_packs_for_testing.sh b/Tests/scripts/prepare_content_packs_for_testing.sh
index 5752a09e7..26b9c42ad 100644
--- a/Tests/scripts/prepare_content_packs_for_testing.sh
+++ b/Tests/scripts/prepare_content_packs_for_testing.sh
@@ -30,8 +30,14 @@ CONTENT_FULL_TARGET_PATH="$GCS_BUILD_BUCKET/$BUILD_BUCKET_PATH/content"
 BUCKET_FULL_TARGET_PATH="$GCS_BUILD_BUCKET/$BUILD_BUCKET_PATH"
 
 # ====== PRODUCTION CONFIGURATION ======
-GCS_MARKET_BUCKET="marketplace-dist"
-SOURCE_PATH="content"
+GCS_MARKET_BUCKET="marketplace-dist-dev"
+SOURCE_PATH="upload-flow/builds/$CIRCLE_BRANCH/$CIRCLE_BUILD_NUM/content"
+
+# ====== UPDATING TESTING BUCKET ======
+echo "Copying production bucket files at: gs://marketplace-dist/content to testing bucket at path: gs://marketplace-dist-dev/$SOURCE_PATH ..."
+gsutil -m cp -r "gs://marketplace-dist/content" "gs://marketplace-dist-dev/$SOURCE_PATH" > "$CIRCLE_ARTIFACTS/logs/Prepare Content Packs For Testing.log" 2>&1
+echo "Finished copying successfully."
+# ====== UPDATING TESTING BUCKET ======
 
 echo "Copying master files at: gs://$GCS_MARKET_BUCKET/$SOURCE_PATH to target path: gs://$CONTENT_FULL_TARGET_PATH ..."
 gsutil -m cp -r "gs://$GCS_MARKET_BUCKET/$SOURCE_PATH" "gs://$CONTENT_FULL_TARGET_PATH" > "$CIRCLE_ARTIFACTS/logs/Prepare Content Packs For Testing.log" 2>&1
@@ -56,7 +62,7 @@ else
   # In Upload-Flow, we exclude test-pbs in the zipped packs
   REMOVE_PBS=true
   BUCKET_UPLOAD_FLOW=true
-  GCS_PRIVATE_BUCKET="marketplace-dist-private"
+  GCS_PRIVATE_BUCKET="marketplace-dist-private-test"
   if [ -n "${FORCE_PACK_UPLOAD}" ] && [ -n "${PACKS_TO_UPLOAD}" ]; then
     # In case the workflow is force upload, we override the forced packs
     echo "Force uploading to production the following packs: ${PACKS_TO_UPLOAD}"
diff --git a/Tests/scripts/slack_notifier.py b/Tests/scripts/slack_notifier.py
index 6dcff82f8..9c575d060 100644
--- a/Tests/scripts/slack_notifier.py
+++ b/Tests/scripts/slack_notifier.py
@@ -285,7 +285,7 @@ def slack_notifier(build_url, slack_token, test_type, env_results_file_name=None
     branch_name_reg = re.search(r'\* (.*)', branches)
     branch_name = branch_name_reg.group(1)
 
-    if branch_name == 'master':
+    if branch_name == 'upload-metadata':
         logging.info("Extracting build status")
         if test_type == UNITTESTS_TYPE:
             logging.info("Starting Slack notifications about nightly build - unit tests")
@@ -313,7 +313,7 @@ def slack_notifier(build_url, slack_token, test_type, env_results_file_name=None
         slack_client = SlackClient(slack_token)
         slack_client.api_call(
             "chat.postMessage",
-            json={'channel': 'dmst-content-team',
+            json={'channel': 'dmst-bucket-upload',
                   'username': 'Content CircleCI',
                   'as_user': 'False',
                   'attachments': content_team_attachments}
diff --git a/Utils/trigger_force_upload_pack_to_production.sh b/Utils/trigger_force_upload_pack_to_production.sh
index 2571b1d9f..e79010041 100755
--- a/Utils/trigger_force_upload_pack_to_production.sh
+++ b/Utils/trigger_force_upload_pack_to_production.sh
@@ -12,7 +12,7 @@ trigger_build_url="https://circleci.com/api/v2/project/github/demisto/content/pi
 
 post_data=$(cat <<-EOF
 {
-  "branch": "master",
+  "branch": "upload-metadata",
   "parameters": {
     "force_pack_upload": "true",
     "packs_to_upload": "${_packs}"
diff --git a/Utils/trigger_upload_packs_to_production.sh b/Utils/trigger_upload_packs_to_production.sh
index 6312c257c..aabe87f77 100755
--- a/Utils/trigger_upload_packs_to_production.sh
+++ b/Utils/trigger_upload_packs_to_production.sh
@@ -11,7 +11,7 @@ trigger_build_url="https://circleci.com/api/v2/project/github/demisto/content/pi
 
 post_data=$(cat <<-EOF
 {
-  "branch": "master",
+  "branch": "upload-metadata",
   "parameters": {
     "bucket_upload": "true"
   }
